plugin-server/src/kafka/batch-consumer.ts

```diff
@@ -76,9 +76,8 @@ export const startBatchConsumer = async ({
         ...connectionConfig,
         'group.id': groupId,
         'session.timeout.ms': sessionTimeout,
-        // We disable auto commit and rather we commit after one batch has
-        // completed.
-        'enable.auto.commit': false,
+        'enable.auto.commit': autoCommit,
```

Comment:
`startSessionRecordingEventsConsumerV1` is not specifying `autoCommit`, so this will switch the value to `true` and change the consumer's behavior.

Seeing as this setting is fundamental, I'd rather remove the default value and force consumers to specify the value, WDYT?